molgenis-navigator/src/main/java/org/molgenis/navigator/delete/ResourceDeleteJobConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.molgenis.navigator.delete;
```

Comment:
Can you put the job related classes in an `org.molgenis.navigator.delete.job` package for consistency with copy and download?